repose-aggregator/components/services/http-client-service/http-client-service-api/src/main/resources/META-INF/schema/config/http-connection-pool.xsd

```diff
@@ -239,14 +239,22 @@
             </xs:annotation>
         </xs:attribute>
 
-        <xs:attribute name="chunked-encoding" type="xs:boolean" use="optional" default="true">
+        <xs:attribute name="chunked-encoding" type="chunkedEncodingType" use="optional" default="true">
             <xs:annotation>
                 <xs:documentation>
                     <html:p>
                         Determines whether or not this client should send chunked data when forwarding requests.
                     </html:p>
                     <html:p>
-                        WARNING: Setting this attribute will cause repose to attempt to retrieve the
+                        If set to true, chunked data will always be sent.
+                        If set to false, chunked data will never be sent.
+                        If set to auto, chunked data will only be sent if the original request to Repose was chunked.
+                        Note that the auto setting only has meaning for the default connection pool, and only when
+                        requests are forwarded to the origin service. A value of auto for non-default pools will
+                        behave identically to a value of true.
```

Comment:
Ah. So indeed, it appears that only requests being made to the origin service check that flag. I hadn't really looked at the code, just the config, so I assumed chunked encoding applied to all requests. Oops. Good catch.
So my point 1 is moot. Although, that totally _could_ mean something to pools other than the default pool. Something to watch out for in the future, I guess.